.eslintrc

```diff
@@ -1,20 +1,22 @@
 {
-  "ecmaFeatures": {
-    "jsx": true,
-    "modules": true
-  },
+  "extends": "eslint-config-airbnb",
   "env": {
     "browser": true,
     "mocha": true,
     "node": true
   },
-  "parser": "babel-eslint",
   "rules": {
-    "quotes": [2, "single"],
-    "strict": [2, "never"],
     "react/jsx-uses-react": 2,
     "react/jsx-uses-vars": 2,
-    "react/react-in-jsx-scope": 2
+    "react/react-in-jsx-scope": 2,
+
+    // Temporary disabled for src/index.js
+    // with following by https://github.com/eslint/eslint/issues/2087
+    "block-scoped-var": 0,
```

Comment:
Can this be because of lack of `parser`? It used to lint fine for me. Anyway, I'd appreciate you changing all `export X from './x'` to `export { X as default } from './x` because TC39 seems to have decided not to advance that proposal.